.github/workflows/nightly.yml

```
@@ -34,796 +36,595 @@ jobs:
+        # The live matrix is Windows-only and its lifecycle/retry scripts use
+        # Bash syntax. Pinning the shell avoids PowerShell parsing them.
+        shell: bash
     env:
-      # Each authenticated lane selects its namespace graph explicitly. Keeping
-      # Web in the matrix preserves the historical default while the separate
-      # Android lane proves the optional package/auth path against live gRPC.
       NOTEBOOKLM_BACKEND: ${{ matrix.backend }}
-      # Pin Playwright's browser install dir to a single workspace-relative
-      # path on every OS — see test.yml for the rationale (sidesteps the
-      # actions/cache@v6 Windows dual-path restore flake).
+      NOTEBOOKLM_HOME: ${{ runner.temp }}/notebooklm-home
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)